<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExitWidget</class>
 <widget class="QWidget" name="ExitWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Angell Medical Equipment Ltd.,&lt;/p&gt;&lt;p&gt;ShenZhen, China.&lt;/p&gt;&lt;p&gt;Copy Right (2014-2015).&lt;/p&gt;&lt;p&gt;&lt;a href=&quot;www.angell-med.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;www.angell-med.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;Product: DR Image Station&lt;/p&gt;&lt;p&gt;Version: 1.0&lt;/p&gt;&lt;p&gt;Author: Mangen Zhu&lt;/p&gt;&lt;p&gt;E-mail: &lt;a href=&quot;mailto:zhumangen@163.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;zhumangen@163.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="openExternalLinks">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>320</width>
           <height>300</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../DicomImageStation_2/resources.qrc">:/images/splash.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="lockStationButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Lock Station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="exitStationButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Exit Station</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="shutDownButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Shut Down</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="rebootButton">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>16</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;微软雅黑&quot;;</string>
       </property>
       <property name="text">
        <string>Reboot</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../DicomImageStation_2/resources.qrc"/>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
